### DISCUSSION TITLE
Stash Color Configuration based on App

### DISCUSSION THREAD

**Discussion by BuchholzTim** [rxhanson/RectanglePro-Community#313] [Ideas]
It would be really great, if we could somehow define a pre-defined color per application for the colored stash-tab (for example: set `Teams` to always be stashed behind a blue stash-tab).

Currently the color is selected at random, as far as I can tell, which makes it hard to remember which stash corresponds to which application.

**Comment by rxhanson** [comment]
Thanks for the feedback. I started working on this prior to the last release and didn't have a chance to finish it yet, so it'll go in an upcoming release.

**Comment by MrLepage** [comment]
Same need on my side.

I use Stash to set aside all my chat apps that I don't need to have constantly on screen but want to keep easily accessible.

Currently, I place Slack, Messenger, Messages, and ChatGPT all on the left side.

At the moment, it's impossible to know which app is behind which tab...

One solution to test could be extracting the dominant color of the app icon and assigning this color to the tab.

Another solution could be writing the name of the app along the tab.

**Comment by MrLepage** [comment]
It might also be good to consider the width of the tabs when calculating the positioning of the windows to avoid having the tabs overlap the windows.
